python-backend/app/api/users/core/models/idir_user_detail.py

```diff
@@ -48,31 +48,31 @@ def find_by_idir_username(cls, idir_username):
             return None
 
     @validates('bcgov_guid')
-    def validate_status_code(self, key, bcgov_guid):
+    def validate_bcgov_guid(self, key, bcgov_guid):
```

Comment:
Do these redefined  functions get used anywhere yet?

Reply:
validates methods are only used by the model they're defined on, or by the tests for them